pkgs/top-level/python-packages.nix

```diff
@@ -2458,6 +2458,8 @@ self: super: with self; {
 
   chromadb = callPackage ../development/python-modules/chromadb { };
 
+  chromadb_0 = callPackage ../development/python-modules/chromadb/0.nix { };
```

Comment:
We don't do versioned attributes in `python3Packages`!

Reply:
Sorry. I knew that this should be avoided, but could be done when necessary. It seemed legit in this case.
Is there a reference describing what would be qualified as an "exceptional case"?

Reply:
I don't think so. But one necessary condition should be that it's used more than once across nixpkgs.